features/networking/service.feature

```diff
@@ -441,16 +441,18 @@ Feature: Service related networking scenarios
   Scenario: Taint node with too small MTU value
     Given the default interface on nodes is stored in the :default_interface clipboard
```

Comment:
nit - node is singular here

Reply:
nit, ok to ignore

Reply:
Thanks Arti. As discussed offline, since its not a line I touched in this PR, I would like to not change it, as it would widen the scope of the PR, having to update the corresponding step definition. There are 16 other tests I can see using this same `incorrect` plural.